### ISSUE TITLE
Attachments not opening

### ISSUE THREAD

**Issue by MyWay** [keepassxreboot/keepassxc#920]
When setting up a new item or when editing one, under advanced section you can add one or more attachments.

## Expected Behavior

Once the attachment has been saved you should be able to open it by double click or clicking on open.

## Current Behavior

Attachment does not launch at all, while it should be launched on the default/favourite OS application set.

## Steps to Reproduce (for bugs)

1. Add an attachment to an existing item, or on a new one
2. Save
3. Try to open it

## Context

I'm trying to have attachments for some items.

## Debug Info

KeePassXC - 2.2.0
Revision: 9a7e6850d6b79c47c2d46d99fdaf433c2ff13716

Libraries:
- LIBS

Operating system: Ubuntu Core 16
CPU architecture: x86_64
Kernel: linux 4.10.0-33-generic

Enabled extensions:
- KeePassHTTP
- Auto-Type
- YubiKey

I used .snap to install keepassxc.

**Comment by droidmonkey** [comment]
This is likely a snap issue...

**Comment by droidmonkey** [comment]
The error that occurs when running in the snap is:
```
dbus-send: /snap/keepassxc/x1/lib/x86_64-linux-gnu/libdbus-1.so.3: version `LIBDBUS_PRIVATE_1.10.6' not found (required by dbus-send)
```

**Comment by droidmonkey** [comment]
This was fixed in 2.2.2 and the fix is the same for the URL opening.
